File: JSBASIC/100_traffic_signal.js
// 100	100_traffic_signal.js	Display the action (Stop/Ready/Go) for a traffic signal color using switch.

let color = "green";
console.log("Traffic Signal Color:", color);
switch (color.toLowerCase()) {
    case "red":    console.log("Action: STOP"); break;
    case "yellow": console.log("Action: READY / GET SET"); break;
    case "green":  console.log("Action: GO"); break;
    default:       console.log("Invalid signal color.");
}

File: JavaBasic/100_traffic_signal.java
// 100	100_traffic_signal.java	Display the action (Stop/Ready/Go) for a traffic signal color using switch.
public class _100_traffic_signal {
    public static void main(String[] args) {
        String color = "green";
        switch (color.toLowerCase()) {
            case "red":    System.out.println("STOP"); break;
            case "yellow": System.out.println("READY / GET SET"); break;
            case "green":  System.out.println("GO"); break;
            default:       System.out.println("Invalid signal color.");
        }
    }
}
